applications/vanilla/views/discussion/helper_functions.php

```diff
@@ -85,11 +85,17 @@ function writeComment($comment, $sender, $session, $currentOffset) {
         $sender->EventArguments['Permalink'] = $permalink;
 
         // Needed in writeCommentOptions()
-        if ($sender->data('Discussion', null) === null) {
+        $discussion = $sender->data('Discussion', null);
+        
+        if ($discussion === null) {
             $discussionModel = new DiscussionModel();
             $discussion = $discussionModel->getID($comment->DiscussionID);
             $sender->setData('Discussion', $discussion);
         }
+        
+        if (($discussion->InsertUserID ?? false) == $comment->InsertUserID) {
+            $cssClass .= ' original-poster';
```

Comment:
Could we change this class name up to be `isOriginalPoster` or `isOP`? We are trying to move to a convention of `is{Something}` for our modifier CSS classes like this.

Reply:
👍  `isOriginalPoster` is less ambiguous